pkgs/tools/networking/airgeddon/default.nix

```
@@ -0,0 +1,126 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+  # Required
+, aircrack-ng
+, bash
+, gawk
+, iproute2
+, iw
+, pciutils
+, procps
+, tmux
+  # Optionals
+  # Missing in nixpkgs: beef, hostapd-wpe, asleap
+, bully
+, crunch
+, dhcp
+, dnsmasq
+, ettercap
+, hashcat
+, hcxdumptool
+, hcxtools
+, hostapd
+, lighttpd
+, mdk4
+, nftables
+, openssl
+, pixiewps
+, reaverwps-t6x # Could be the upstream version too
+, wireshark-cli
+  # Support groups
+, supportWpaWps ? true # Most common use-case
+, supportHashCracking ? false
+, supportEvilTwin ? false
+}:
+with lib;
+let
+  deps = [
+    aircrack-ng
+    bash
+    gawk
+    iproute2
+    iw
+    pciutils
+    procps
+    tmux
+  ] ++ (lists.optionals supportWpaWps [
+    bully
+    pixiewps
+    reaverwps-t6x
+  ]) ++ (lists.optionals supportHashCracking [
+    crunch
+    hashcat
+    hcxdumptool
+    hcxtools
+    wireshark-cli
+  ]) ++ (lists.optionals supportEvilTwin [
+    dhcp
+    dnsmasq
+    ettercap
+    hostapd
+    lighttpd
+    openssl
+    mdk4
+    nftables
+  ]);
+in
+stdenv.mkDerivation rec {
+  pname = "airgeddon";
+  version = "11.01";
+
+  src = fetchFromGitHub {
+    owner = "v1s1t0r1sh3r3";
+    repo = "airgeddon";
+    rev = "v${version}";
+    sha256 = "3TjaLEcerRk69Ys4kj7vOMCRUd0ifFJzL4MB5ifoK68=";
+  };
+
+  strictDeps = true;
+  buildInputs = [ makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ makeWrapper ];
```
This is supposed to only be ran in the build machine, so nativeBuildInputs is the correct choice afaik.